wave_docs/api.mdx

```diff
@@ -100,3 +284,275 @@ Provides basic information about the service status.
     }
 }
 ```
+
+## Metrics APIs based on SurrealDB
+
+These APIs provide usage (builds and pulls) metrics of Wave per IP and user.
+These APIs required basic authentication, so you need to provide username and password while calling these APIs.
+
+All Metrics API endpoints have these three common optional query parameters:
+
+| Name      | Description                                                                                   | sample Value |
+|-----------|-----------------------------------------------------------------------------------------------|--------------|
+| startDate | Format: `yyyy-mm-dd`, The data of this date will be included in computing the response.       | 2024-02-29   |
+| endDate   | Format: `yyyy-mm-dd`, The data of this date will be included in computing the response.       | 2024-02-29   |
+| limit     | Integer from `0` to `1000` and default is `100`. It limits the number of records in response. | 10           |
+
+Note: If you only provide startDate then endDate will be today's date. 
+
+### Build Metrics APIs
+
+These APIs are used to get the metrics about the container builds by Wave.
+
+All build metrics API endpoints have one common optional query parameter:
+
+| Name    | Description                                                                                                   | sample Value |
+|---------|---------------------------------------------------------------------------------------------------------------|--------------|
+| success | `true` or `false` is allowed and its used to get the metrics of successful container builds performed by Wave | true         |
```

Comment:
Default value may be more useful than a sample value.

Retrieve the metrics of successful container builds performed by Wave (`true` or `false`).